### DISCUSSION TITLE
how to use transformations?

### DISCUSSION THREAD

**Discussion by FynnJason** [coil-kt/coil#1030] [Q&A]
iv_4.load(IMAGE_URL){
            placeholder(R.mipmap.ic_launcher)
            error(R.mipmap.ic_launcher_round)
            transformations(BlurTransformation(applicationContext))
            transformations(CircleCropTransformation())
        }

bus noly use transformations(CircleCropTransformation()),how can i use BlurTransformation and CircleCropTransformation together?

**Comment by colinrtwhite** [comment]
`transformations` accepts an array of transformations: `transformations(CircleCropTransformation(), BlurTransformation(applicationContext))`
